### ISSUE TITLE
gRPC error

### ISSUE THREAD

**Issue by aleenakvk** [bendangnuksung/mrcnn_serving_ready#33]
Received gRPC Error: StatusCode.INVALID_ARGUMENT - input size does not match signature: 1!=3 len({inputs}) != len({input_anchors,input_image,input_image_meta}). Sent extra: {inputs}. Missing but required: {input_anchors,input_image,input_image_meta}.

I have got an error like this while serving. So is there any method to avoid this error without having any preprocessing on clientAPI

**Comment by smujiang** [comment]
@aleenakvk I got the same error, have you found a solution?

**Comment by aleenakvk** [comment]
> @aleenakvk I got the same error, have you found a solution?

Yes. My client API was already built by another software TensorFlow serving API. In client API they have asked for exactly one input. But according to the code in here it is sending 3 inputs and the client-side should be ready to accept this input. But in my case, I couldn't change the Client side. So I have changed the server-side. i.e, There is a pre-trained TensorFlow serving model for MaskRCNN is available on the TensorFlow API server website. So I have downloaded that and changed some architecture in accordance with my objective. So client-side and server-side, inputs and outputs got matched. 

This solution was specific to my case. I am not sure whether this work for your case. I hope I could provide some insights.
